src/ray/util/pipe_logger.cc

```diff
@@ -338,35 +294,27 @@ RedirectionFileHandle CreateRedirectionFileHandle(
   };
   auto flush_fn = [logger,
                    stream_redirect_opt = stream_redirect_opt,
-                   std_stream_fd = std_stream_fd]() {
+                   std_stream_sink = std_stream_sink]() {
```

Comment:
move?

Reply:
It's intentionally copied here.

I made a mistake when I was writing the code --- I switch the order for `write_fn` and `flush_fn`, then use-after-move. I feel like move semantics applies in most of the places, but here, before we enable clang-tidy in our CI (which checks invalid usage) I would be conservative.